e2e/tests/manageCases/caseProgression/checkApplication/checkApplication.spec.ts

```diff
@@ -19,6 +21,8 @@ test.describe("Check Application task for DA Citizen case tests. @manageCases",
     await CheckApplication.checkApplication({
       page: page,
       accessibilityTest: false,
+      yesNoSendToGateKeeper: true,
```

Comment:
i think we might want to have a think about these tests and what we name them if they're being built on as part of a longer e2e journey.